### DISCUSSION TITLE
wireguard status show peer name

### DISCUSSION THREAD

**Discussion by locnos** [eduardogsilva/wireguard_webadmin#45] [Ideas]
Hi, would it be possible to show the same name of the peer with which it was created? In wireguard status it shows an encrypted id, thanks
![Captura desde 2024-12-04 22-36-06](https://github.com/user-attachments/assets/63a0d55b-c620-4a04-852a-16877ed719cc)
![Captura desde 2024-12-04 22-34-17](https://github.com/user-attachments/assets/b1394057-9813-493a-af43-6d9b055511d3)

Another issue, long names give error when importing into wireguard client

![Captura desde 2024-12-05 08-54-44](https://github.com/user-attachments/assets/9243a7d6-9ecc-4650-abee-67c4d67b7d8b)

On Android, when I download the peer conf, it downloads with the extension .txt
peer_6482261d-a3ce-403c-a00d-bc5b0e259a64.conf.txt
On Linux or Windows it's ok
peer_6482261d-a3ce-403c-a00d-bc5b0e259a64.conf

**Comment by eduardogsilva** [comment]
Hello there!

Sorry for the "very late response". 

If you edit the peer and remove the Name, it will show the first 16 characters of the wireguard peer name... The idea of the name field, is just to keep it easier to identify.

Regarding the file name, if you rename the file, it will allow to be imported. 

To make it easier, I could change the name convention for the peer file for something like:

peer.conf
peer_<unixtimestamp>.conf (when peer was created)
or also maybe peer_<pk>.conf 


Regarding the .txt in the end, I believe that maybe it's something on your android version and/or browser.

Cheers.

**Comment by eduardogsilva** [comment]
Hello @locnos I've changed the default filename structure and I believe that the problem should be fixed now. 

Please let me know if the issue still persist.
